src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -4,11 +4,25 @@
 
 from __future__ import absolute_import, division, print_function
 
+import os
 import threading
 
 from cryptography.hazmat.bindings._openssl import ffi, lib
 
 
+@ffi.callback("int (*)(unsigned char *, int)", error=-1)
```

Comment:
Things will work fine without it, but I'm curious why we're returning -1 here in case of error? 0 is what the old code returned.